docker-entrypoint.sh

```diff
@@ -35,12 +35,26 @@ drop_privs_cmd() {
     fi
 }
 
+copy_plugins_if_required() {
+  echo "Enabling required built-in plugins"
+  for TARGET_PLUGIN in $(echo "$ENABLE_BUILT_IN_PLUGINS" | grep -o -e "[^;]*"); do
+    echo "Moving $TARGET_PLUGIN to plugin directory"
+    if cp "${FLINK_HOME}/opt/${TARGET_PLUGIN}" "${FLINK_HOME}/plugins"; then
```

Comment:
Copying the plugins directly into Flink's plugins folder is unfortunately not supported yet. You need to create a subfolder for each plugin. See also https://ci.apache.org/projects/flink/flink-docs-master/ops/plugins.html#isolation-and-plugin-structure

Reply:
Looking at the plugins currently shipped in the `opt` directory, I don't see any that need to be bundled with another jar in order to work. Therefore, are you ok with me placing each plugin in a directory with the same name as the jar? So: `flink/plugins/flink-s3-fs-presto-1.8.1/flink-s3-fs-presto-1.8.1.jar` ?

Reply:
I've pushed this change - let me know what you think